lib/widgets/emoji_reaction.dart

```diff
@@ -580,8 +580,7 @@ class EmojiPickerListEntry extends StatelessWidget {
       child: Padding(
         padding: const EdgeInsets.symmetric(horizontal: 8),
         child: Row(spacing: 4, children: [
-          if (glyph != null)
-            Padding(
+          Padding(
               padding: const EdgeInsets.all(10),
               child: glyph),
```

Comment:
This adds horizontal padding (`const EdgeInsets.all(10)`). We don't want to do that, as I mentioned in https://github.com/zulip/zulip-flutter/pull/1759#pullrequestreview-3076631735.

I suggest keeping `null` for `glyph` in the text-emoji case, and adding an `else` here that makes a `SizedBox(height: _emojiSize + 10 + 10)`. Similarly in the other case.